frontend/class-twitter.php

```diff
@@ -158,6 +158,9 @@ protected function description() {
 		if ( is_singular() ) {
 			$meta_desc = $this->single_description();
 		}
+		elseif ( is_home() && ! is_front_page() ) {
```

Comment:
`is_posts_page()` was correct, but documented wrong. Updated documentation and switched PR to use it.